pkg/hubble/observer/local_observer.go

```diff
@@ -127,9 +127,13 @@ nextEvent:
 
 		ev, err := s.payloadParser.Decode(monitorEvent)
 		if err != nil {
-			if !errors.Is(err, parserErrors.ErrUnknownEventType) {
-				// Debug event types MessageTypeDebug and MessageTypeCapture are treated as invalid type.
-				// To avoid spamming debug log, silence them until the parser for them is implemented.
+			switch {
+			case errors.Is(err, parserErrors.ErrUnknownEventType),
+				errors.Is(err, parserErrors.ErrEventSkipped):
+				// silently ignore those kind of errors
+			default:
+				// silently ignore perf ring buffer events with unknown types,
+				// since they are not intended for us (e.g. MessageTypeRecCapture)
 				if !parserErrors.IsErrInvalidType(err) {
 					s.log.WithError(err).WithField("event", monitorEvent).Debug("failed to decode payload")
 				}
```

Comment:
nit: could handle the invalid type case in the `switch` directly

```suggestion
			switch {
			// silently ignore those kind of errors
			case errors.Is(err, parserErrors.ErrUnknownEventType),
				errors.Is(err, parserErrors.ErrEventSkipped),
				// silently ignore perf ring buffer events with invalid types,
				// since they are not intended for us (e.g. MessageTypeRecCapture)
				parserErrors.IsErrInvalidType(err):
			default:
				s.log.WithError(err).WithField("event", monitorEvent).Debug("failed to decode payload")
```

Reply:
Good suggestion, applied!